draft-ietf-rtcweb-jsep.xml

```diff
@@ -205,11 +205,11 @@
           When <xref target="RFC8829"/> was published, inconsistencies regarding BUNDLE
           <xref target="RFC8843"/> operation were identified with regard to
```

Comment:
I'm guessing we want to leave this one as RFC8843?

Reply:
I think this one we do want to leave alone because it is referring to the "old" version.

Reply:
OK. I will update accordingly and merge.

Reply:
PTAL at the updated text for this PR

Reply:
@seanturner